authority/provisioner/options.go

```diff
@@ -55,17 +54,11 @@ func (o *Options) GetSSHOptions() *SSHOptions {
 }
 
 // GetWireOptions returns the SSH options.
-func (o *Options) GetWireOptions() (*wire.Options, error) {
+func (o *Options) GetWireOptions() *wire.Options {
```

Comment:
ditto

Reply:
Changed this to return an error in 194341e520f2c7f0f9049ab65fa42b8898101d0e